src/selfplay/game.cc

```diff
@@ -45,7 +47,7 @@ SelfPlayGame::SelfPlayGame(PlayerOptions player1, PlayerOptions player2,
   }
 }
 
-void SelfPlayGame::Play(int white_threads, int black_threads) {
+void SelfPlayGame::Play(int white_threads, int black_threads, bool use_resign) {
```

Comment:
For my taste `allow_resign` name would fit better, but I'm ok with keeping it.

Reply:
Perhaps `disable_resign`? Since what we're doing is an override, the variable should be true when we're taking positive action

Reply:
Done, ish, please re-review